controllers/comments.controllers.js

```diff
@@ -34,3 +35,16 @@ exports.postNewComment = (req, res, next) => {
       next(err);
     });
 };
+
+exports.deleteComment = (req, res, next) => {
+    const { comment_id } = req.params;
+    const deleteCommentQuery = removeComment(comment_id)
+    const commentIDExistenceQuery = checkCommentIDexists(comment_id)
+
+    Promise.all([deleteCommentQuery, commentIDExistenceQuery])
```

Comment:
We could rely on the psql query to tell us if something has been successfully deleted or not rather than running a separate check that it exists. If we return in our sql query we will have something to check. Remember not to send what's returned back to the user in the controller though.